src/test/scala/com/gu/anghammarad/models/SerializationTest.scala

```diff
@@ -0,0 +1,107 @@
+package com.gu.anghammarad.models
+
+import io.circe.Json
+import io.circe.parser._
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.io.Source
+
+class SerializationTest extends FreeSpec with Matchers {
+
+  "parseAllMappings" - {
+
+    "will parse all mappings when given a valid string" in {
+      val validJsonString = Source.fromURL(getClass.getResource("/config.json")).mkString
+
+      val expectedResult = List(
+        Mapping(List(Stack("stack1")), List(EmailAddress("stack1.email"), HangoutsRoom("stack1.room"))),
+        Mapping(List(Stack("stack1"), App("app1")), List(EmailAddress("app1.email"), HangoutsRoom("app1.room"))),
+        Mapping(List(AwsAccount("123456789")), List(EmailAddress("awsAccount.email")))
+      )
+
+      Serialization.parseAllMappings(validJsonString).get shouldEqual expectedResult
+    }
+
+    "will return a failure if there are no mappings" in {
+      val noMappingsString = """{"emailDomain":"example.com"}"""
+
+      Serialization.parseAllMappings(noMappingsString).isFailure shouldEqual true
+    }
+
+    "will return a failure if the string is not valid json" in {
+      val brokenJsonString =
+        """{
```

Comment:
This is very hard to read! Can it be indented nicely? Makes it hard to see in what sense this is "broken". If it's just invalid JSON then would a very short invalid string by a reduced test case?